astropy/nddata/utils.py

```diff
@@ -69,6 +75,9 @@ def overlap_slices(large_array_shape, small_array_shape, position, mode="partial
         otherwise an `~astropy.nddata.utils.PartialOverlapError` is
         raised.  In all modes, non-overlapping arrays will raise a
         `~astropy.nddata.utils.NoOverlapError`.
+    limit_rounding_method : {'ceil', 'round' or 'floor'}
+        Allows the user to choose the rounding method of their own choice when trimming the pixel indices.
+        Has a default value of "ceil", if not altered.
```

Comment:
```suggestion
        Has a default value of "ceil".
```
"if not altered" is a bit redundant here.